code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -1,3 +1,7 @@
+
+/datum/action/xeno_action/activable/pounce/predalien/pre_pounce_effects(atom/A)
```

Comment:
this is passing an arg that doesn't exist on base (also is one letter)

Reply:
bweh